index.html

```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
 </div>
 
  <div class="card-works">
```

Comment:
- [ ] You did great implementing the about section :+1: :+1:  but I think there are some improvements you need to make here before moving to the next milestone. 
- Following the best practices, your entire work section should be wrapped in a `section` tag. this is for semantics purposes as it helps in improving your website SEO. :+1: :smile: